packages/itwin/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -61,13 +63,34 @@ MeasureAreaToolModel
     this._enableSheetMeasurements = enableSheetMeasurements;
   }
 
+  private async updateDrawingTypeCache() {
+    this._sheetChangeListener();
+    const sheetIds = [];
+    if (this._enableSheetMeasurements) {
+      for (const viewport of IModelApp.viewManager) {
+        if (viewport.view.classFullName === "BisCore:SheetViewDefinition") {
```

Comment:
This will not work for subclasses of SheetViewDefinition. Check for `instanceof SheetViewDefinition` instead

Reply:
or `view.isSheetView()`